notebook/tests/selenium/test_deletecell.py

```diff
@@ -3,10 +3,9 @@ def cell_is_deletable(nb, index):
     JS = 'return Jupyter.notebook.get_cell({}).is_deletable();'.format(index)
     return nb.browser.execute_script(JS)
 
-def delete_cell(notebook, index):
-    notebook.focus_cell(index)
-    notebook.to_command_mode
-    notebook.current_cell.send_keys('dd')
+def remove_cells(notebook):
```

Comment:
Let's call this `remove_all_cells`, just to be clear what it's doing.

Reply:
Also, I think the implementation could be more efficient. If you look at what `notebook.index()` does, it's actually querying the browser for the full list of cells to find out where the given cell is. We don't need to know that, we just need to know how *many* cells to delete. Then we can do it by deleting the first cell N times.

Reply:
makes sense. The current way is quite inefficient .